test/command/list_test.rb

```diff
@@ -3,23 +3,41 @@
 
 describe Licensed::Command::List do
   let(:config) { Licensed::Configuration.new }
+  let(:source) { TestSource.new }
   let(:command) { Licensed::Command::List.new(config) }
+  let(:fixtures) { File.expand_path("../../fixtures", __FILE__) }
 
-  it "lists dependencies for all source types" do
-    out, = capture_io { command.run }
-    config.sources.each do |s|
-      assert_match(/#{s.type} dependencies:/, out)
+  before do
+    config.apps.each do |app|
+      app.sources.clear
+      app.sources << source
+    end
+  end
+
+  each_source do |source_type|
```

Comment:
same as above here - using a configuration fixture seemed like the best approach here